tools/windows-node-installer/pkg/cmd/flags.go

```diff
@@ -0,0 +1,7 @@
+package cmd
```

Comment:
Why is this needed?

Reply:
This says the functionalities of the binary. It is in the grey area of not implemented. I think it might just go with the Flag PR then?